sql/2021/accessibility/skip_links.sql

```diff
@@ -0,0 +1,21 @@
+#standardSQL
+# % of pages having skip links
+CREATE TEMPORARY FUNCTION getEarlyHash(payload STRING)
```

Comment:
Nit: This one should be straightforward to implement using `JSON` functions. Was it the hyphens in the property name that threw you off?